client/my-sites/themes/index.web.js

```diff
@@ -6,7 +6,8 @@ import userFactory from 'lib/user';
 import { makeLayout } from 'controller';
 import { makeNavigation, siteSelection } from 'my-sites/controller';
 import { singleSite, multiSite, loggedOut } from './controller';
-import { getSubjects } from './theme-filters.js';
+import { getSubjects } from './theme-filters';
+import validateFilters from './validate-filters';
```

Comment:
Yeah :-/ Okay, so let's keep it in there for now, but maybe add a comment at the top of that file to the effect of this discussion :-)